reporter-web-app/src/components/RuleViolationsTable.js

```diff
@@ -162,31 +162,37 @@ class RuleViolationsTable extends React.Component {
                 onFilter: (value, webAppRuleViolation) => {
                     const webAppPackage = webAppRuleViolation.package;
 
-                    if (value === 'excluded') {
-                        return webAppPackage.isExcluded;
-                    }
+                    if (webAppPackage) {
```

Comment:
Could this be simplified as follows? (maybe it's out of scope of this commit)
```
onFilter: (value, webAppRuleViolation) => {
  const webAppPackage = webAppRuleViolation.package;
  const isExcluded = webAppPackage && webAppPackage.isExcluded

  return value === 'excluded' ?: isExcluded : !isExcluded
}
```

or

```
onFilter: (value, webAppRuleViolation) => {
  if (!webAppRuleViolation.hasPackage) return false

  const isExcluded = webAppRuleViolation.package.isExcluded

  return isExcluded && value === 'excluded' || !isExcluded && value === 'included'
}
```

Reply:
Has been implemented in a separate commit on top.